transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/MessagingClientImpl.kt

```diff
@@ -84,6 +85,7 @@ internal class MessagingClientImpl(
     private var connectAuthenticated = false
     private var isStartingANewSession = false
     private var reconfigureAttempts = 0
+    private var _fileAttachmentProfile: FileAttachmentProfile? = null
```

Comment:
Why is there a need to have a private `_fileAttachmentProfile` and a public `fileAttachmentProfile` variables?